src/Avatar.svelte

```diff
@@ -21,9 +23,10 @@
     return avatar.toDataURL();
   }
 
-  $: if (isAddress(source) || isRadicleId(source)) {
+  if (isAddress(source) || isRadicleId(source) || isPeerId(source)) {
```

Comment:
Didn't we have this as reactive because we switched from blockies to image avatars on load?

Reply:
Yeah that was my thinking too, but the initial unresolved value is normally an `Address | URN | Peer ID` and we can check for that and generate the blocky during the `onMount` of `Avatar`.

When e.g. a profile has been resolved we update the `source` prop from the parent component to an image url, and the `Avatar` component updates, without needing reactivity itself.

The only way this could fail is if we go from e.g. an image URL back to a URN, which is not a realistic scenario or possible edge case IMO.